### PR TITLE
Decrease mav rx to ardu baud

### PR DESCRIPTION
**Pull request by JyeSmith**
As suggested by MT, we appear to be dropping bytes on the wire from the FC.  As a result we drop about 1% of the packets because they never actually make it to the Rx.

Reducing the baud looks to eliminate (or greatly reduce) the issue.

image

### CODE REVIEW

**Comment by JyeSmith** [comment]
Added a limit to the max packet rate.  Prof Wez mentioned that at 1000Hz (500Hz RC rate) we will be saturating the TX line.  Confirmed to be correct and at a more sensible 250Hz TX line looks much better.

The bottom RC trace indicates when RC packet are sent from the Rx to FC.
image

File: src/src/rx-serial/SerialMavlink.cpp [hunk 1]
```diff
@@ -91,8 +93,8 @@ uint32_t SerialMavlink::sendRCFrame(bool frameAvailable, bool frameMissed, uint3
     mavlink_msg_rc_channels_override_encode(this_system_id, this_component_id, &msg, &rc_override);
     uint16_t len = mavlink_msg_to_send_buffer(buf, &msg);
     _outputPort->write(buf, len);
-    
-    return DURATION_IMMEDIATELY;
+
+    return MIN_PACKET_INTERVAL;
```
Comment (wvarty) [inline src/src/rx-serial/SerialMavlink.cpp:97]: This method of rate limiting causes major impacts to TQTY on ESP8285 receivers. Need to use a "simpler" approach to check if we should forward the pkt out or not.

**Review by wvarty — CHANGES_REQUESTED** [review]
Rate limiting needs work

**Comment by wvarty** [comment]
Retested after latest changes to the rate limiting.

460k baud after 10mins:
3 x x 460k

230k baud after 10mins:
230k

Seems like a negligible difference in dropped pkt counts... not worth changing?

**Pull request closed without merging**